src/components/Group/Navbar.vue

```diff
@@ -1,38 +1,41 @@
 <template>
-    <q-tabs v-if="shouldBeDisplayed" align="center" color="tertiary" class="mobile-only shadow-14 inset-shadow" hide="icon" slot="navigation">
-        <q-route-tab
-            :to="{name: 'group'}"
-            exact
-            slot="title"
-            name="wall"
-        >{{ $t('GROUP.WALL') }}</q-route-tab>
-        <q-route-tab
-            :to="{name: 'groupDescription'}"
-            exact
-            slot="title"
-            name="description"
-        >{{ $t('GROUP.DESCRIPTION') }}</q-route-tab>
-        <q-route-tab
-            :to="{name: 'groupHistory'}"
-            exact
-            slot="title"
-            name="history"
-        >{{ $t('GROUP.HISTORY') }}</q-route-tab>
+  <div slot="navigation" class="shadow-14 inset-shadow row">
+    <q-tabs color="tertiary" hide="icon" class="col">
+      <q-route-tab
+        :to="{name: 'group'}"
+        exact
+        slot="title"
+        name="wall">
+        {{ $t('GROUP.WALL') }}
+      </q-route-tab>
+      <q-route-tab
+        :to="{name: 'groupDescription'}"
+        exact
+        slot="title"
+        name="description">
+        {{ $t('GROUP.DESCRIPTION') }}
+      </q-route-tab>
+      <q-route-tab
+        :to="{name: 'groupHistory'}"
+        exact
+        slot="title"
+        name="history">
+        {{ $t('GROUP.HISTORY') }}
+      </q-route-tab>
     </q-tabs>
+    <q-btn flat class="bg-tertiary text-white">
+      <q-icon name="fa-ellipsis-v" />
```

Comment:
I was always using the `<i class="fa fa-ellipsis-v"/>` style for icons. Is there an advantage of using the quasar ones?

Reply:
I guess both is fine, `q-icon` seems to be a bit more explicit than the `i` tag.
It provides the same interface for using fontawesome and materialicons, if we need the latter at all: http://quasar-framework.org/components/icons.html#Basic-Usage